core/src/main/java/org/dst/core/operatorImpl/DstListImpl.java

```diff
@@ -35,21 +39,53 @@ public boolean del(String key) {
 
   @Override
   public boolean lput(String key, List<String> value) {
-    throw new NotImplementException("The method is not implemented");
+    if (listMap.containsKey(key)) {
```

Comment:
I believe this will get clearer:
```Java
if (!listMap.containsKey(key)) {
    return Status.KEY_NOT_FOUND;
} 
listMap.get(key).addAll(0, value);
return Status.OK;
```

Reply:
I agree,  I'll refactor the code in a second